### PR TITLE
Added Vimrc and some grammatical changes.

### PR DESCRIPTION
**Pull request by sdanthinne**
I proofread the whole thing and it seems pretty good! I just added some small changes that I thought would be helpful.

### CODE REVIEW

File: mod1/README.md [hunk 1]
```diff
@@ -239,7 +247,7 @@ This is the part where the compiler screams at you for missing a semicolon (I ho
 
 I would highly recommend using additional warning flags. It might seem counter intuitive to purposefully ask the compiler to give you more errors, but it can really help point out when you've screwed up. Here's my personal stash (straight from a `Makefile`), you should look up what they do:
 
-`WARNINGS=-pedantic -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wuninitialized -Wmissing-declarations -Werror=format-security`
+`WARNINGS=-pedantic -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Wmissing-declarations -Werror=format-security`
```
Comment (sfshaw) [inline mod1/README.md:251]: Are these removed since they're included in other flags?

Comment (sdanthinne) [reply mod1/README.md:251]: yeah, they are included within -Wall (https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html)

Comment (sfshaw) [reply mod1/README.md:251]: It looks like init-self is gcc specific. I made sure clang behaves the same way. 👍

File: README.md [hunk 2]
```diff
@@ -93,3 +93,4 @@ Pull requests welcome!
 
 - [Spencer Shaw](https://github.com/sfshaw-calpoly)
 - [Kedwin Chen](https://github.com/kedwinchen)
+- [Sebastien Danthinne](https://github.com/sdanthinne)
```
Comment (sfshaw) [inline README.md:96]: @sdanthinne Would you like a different link?

Comment (sdanthinne) [reply README.md:96]: That's fine

**Review by sfshaw — APPROVED** [review]
Thanks for your contribution, Sebastien!

**Pull request merged**